app/src/main/java/com/github/mobile/GitHubModule.java

```diff
@@ -80,8 +82,11 @@ Retrofit retrofit(Provider<GitHubAccount> accountProvider) {
                 .addInterceptor(new RequestConfiguration(accountProvider))
                 .build();
 
+        JsonAdapter<Milestone> adapter =
+                new Moshi.Builder().add(new DateAdapter()).build().adapter(Milestone.class).serializeNulls();
         Moshi converter = new Moshi.Builder()
                 .add(new DateAdapter())
+                .add(Milestone.class, adapter)
```

Comment:
To allow setting null due_date for milestone

Reply:
Can't you achieve the same result by modifying the current `DateAdapter`?

Reply:
No, default adapter for milestones just ignores null properties.
So there is no other way to send patch request with due_on = null.

Reply:
No, I mean changing the behavior of https://github.com/jonan/ForkHub/blob/master/app/src/main/java/com/github/mobile/api/DateAdapter.java which is already being used.

Reply:
If we set due_on = null, then DateAdapter is not being called by Milestone adapter because it skips null properties.
Yes, we can achieve it only with DateAdapter, if use some date as null. For example,  DateAdapter will replace  01/01/1990  to null. But I am not sure, that it is a better alternative.